x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportDeleteTrainedModelAction.java

```diff
@@ -142,7 +142,7 @@ public ClusterState execute(final ClusterState currentState) {
         });
     }
 
-    private Set<String> getReferencedModelKeys(IngestMetadata ingestMetadata) {
+    static Set<String> getReferencedModelKeys(IngestMetadata ingestMetadata, IngestService ingestService) {
```

Comment:
Unused?

Reply:
it is used also in the transport delete trained model alias class to gather all the referenced models to make sure a model alias is not used by a pipeline